index.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 module.exports = class EmitAllPlugin {
     constructor(opts = {}) {
         this.ignorePattern = opts.ignorePattern || /node_modules/;
+        this.ignoreExternals = opts.externals;
```

Comment:
`this.ignoreExternals` should be given a default value. Since it should be a boolean value, `this.ignoreExternals = !! opts.externals;` could be used if the default value should be `false`.